docs/5.x/reporting-api.md

```diff
@@ -73,7 +73,11 @@ _referrerName==Google,referrerName==Bing;country==IN_
     
 *   **filter\_limit**; defines the number of rows to be returned. 
     * By default, only the top **100 rows** are returned.
-    * Set to -1 to return all rows. 
+    * Set to -1 to return all rows.
+ 
+*   **showMetadata**; defines if the response should include additional metadata (available since Matomo 5.4.0)
+    * By default metadata will be included
+    * Set to 0 to hide metadata
```

Comment:
```suggestion
*   **showMetadata**; defines if the response should include additional metadata (available since Matomo 5.4.0).
    * By default, metadata will be included.
    * Set to 0 to hide metadata.
```

Not 100% sure about those, but matching the punctuation of the previous entry exactly should not be too wrong :thinking: